src/website/apps/blog/tests.py

```
@@ -231,4 +244,675 @@ def test_calendar_endpoint_handles_legacy_quill_data(self):
+        self.assertEqual(link.parent_event, main_event)
+        self.assertEqual(link.side_event, side_event)
+        self.assertTrue(side_event.is_side_event)
+        self.assertTrue(main_event.has_side_events)
+        self.assertFalse(main_event.is_side_event)
+
+    def test_duplicate_side_event_link_is_rejected_at_db_level(self):
+        main_event = self._make_event('Main')
+        side_event = self._make_event('Side')
+        EventSideEvent.objects.create(
+            parent_event=main_event, side_event=side_event)
+        with self.assertRaises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)